math/genvector/inc/Math/GenVector/Cartesian3D.h

```diff
@@ -307,7 +312,5 @@ void Cartesian3D<T>::SetEta(Scalar eta) {
 
 #endif
 
-
-
-
+//#endif // R__HAS_VECCORE
```

Comment:
Sorry, I will cleanup it, it was a test to add directly VecCore triginimetric functions, so I should add it, but at the end it didn't help..